helm/dns-operator-azure/templates/deployment.yaml

```diff
@@ -23,6 +23,7 @@ spec:
       securityContext:
         runAsUser: {{ .Values.pod.user.id }}
         runAsGroup: {{ .Values.pod.group.id }}
+      hostNetwork: true
```

Comment:
should we make it a value with default `true` ?

Reply:
i don't see a real need for that because only we (CAPZ within GS) are using it. And if it runs in container network namespace, we have to fix the identity topic (which isn't implemented yet).

I would keep the `values.yaml` as simple as possible.